tests/perf/pubsub_bulk_subscribe_grpc/pubsub_bulk_subscribe_grpc_test.go

```diff
@@ -0,0 +1,200 @@
+//go:build perf
+// +build perf
+
+/*
+Copyright 2022 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package pubsub_bulk_subscribe_grpc_perf
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/dapr/dapr/tests/perf"
+	"github.com/dapr/dapr/tests/perf/utils"
+	kube "github.com/dapr/dapr/tests/platforms/kubernetes"
+	"github.com/dapr/dapr/tests/runner"
+)
+
+const numHealthChecks = 60 // Number of times to check for endpoint health per app.
+
+const testLabel = "pubsub_bulk_subscribe_grpc"
+
+var tr *runner.TestRunner
+
+func TestMain(m *testing.M) {
+	utils.SetupLogs("pubsub_bulk_subscribe_grpc")
+
+	testApps := []kube.AppDescription{
+		{
+			AppName:           "testapp",
+			DaprEnabled:       true,
+			ImageName:         "perf-pubsub_bulk_subscribe_grpc",
+			Replicas:          1,
+			IngressEnabled:    true,
+			MetricsEnabled:    true,
+			AppPort:           3000,
+			AppProtocol:       "grpc",
+			DaprCPULimit:      "4.0",
+			DaprCPURequest:    "0.1",
+			DaprMemoryLimit:   "512Mi",
+			DaprMemoryRequest: "250Mi",
+			AppCPULimit:       "4.0",
+			AppCPURequest:     "0.1",
+			AppMemoryLimit:    "800Mi",
+			AppMemoryRequest:  "2500Mi",
+			Labels: map[string]string{
+				"daprtest": testLabel + "-testapp",
+			},
+		},
+		{
+			AppName:           "tester",
+			DaprEnabled:       true,
+			ImageName:         "perf-tester",
```

Comment:
nit:
are you aware that we now also support https://github.com/grafana/k6 as a load testing tool?

If so, do you have any concern on using it instead of `perf-tester` ? there are plans to export k6 metrics to a persistent prometheus server to get a holistic view of performance (instead of hardcoded thresholds), so it would be nice to have the new perf tests being created using it.

Reply:
btw: If you need help to create those, please do not hesitate to reach me out

Reply:
Hello @mcandeia, yes, I did see the k6 addition. The publish and bulk-publish tests are already added to Dapr with Fortio, so adding this as Fortio as well for consistency. We can certainly migrate them later (with other Fortio tests) if the metrics help us. Do you have examples of using k6 with gRPC?

Reply:
I haven’t made any, but you can certainly use the examples from documentation https://k6.io/docs/javascript-api/k6-net-grpc/, let me know in case of having issues